.github/workflows/macos-build.yml

```diff
@@ -0,0 +1,73 @@
+name: build
```

Comment:
Probably to be changed in the future with a more descriptive name. No blocking for now

Reply:
yes, I'll change with next update